app/Http/Controllers/TrainingDash.php

```diff
@@ -362,7 +365,8 @@ public function saveTicket(Request $request, $id) {
                 'date' => 'required',
                 'start' => 'required',
                 'end' => 'required',
-                'duration' => 'required'
+                'duration' => 'required',
+                'score' => ['required', 'integer', 'between:1,5']
             ]);
```

Comment:
Please add validation here to ensure that movements is numeric. Per the API errata, movements requires an integer.